### DISCUSSION TITLE
come si fermano le notifiche?

### DISCUSSION THREAD

**Discussion by mfagiolihi-net** [globaleaks/globaleaks-whistleblowing-software#3996] [Q&A]
anche se ho tutte le segnalazioni in stato chiuso continuano ad arrivare avvisi via email al ricevente. 
Le email indicano che ha delle notifiche da gestire,ma sono tutte chiuse come posso fermarle?
![image](https://github.com/globaleaks/GlobaLeaks/assets/155471806/8fba7783-2112-4aae-9293-5164929d360c)

**Comment by evilaliv3** [comment]
Vedo che avete i promemoria abilitati, disabi9litandoli smetterete di ricevere le notifiche che la data di promemoria è passata

Al momento la funzionalità di promemoria è slegata dallo stato di segnalazione (potrebbe esserci una segnalazione chiusa su cui comunque volete essere notificati)

**Comment by mfagiolihi-net** [comment]
grazie per il chiarimento, non si potrebbe migliorare la documentazione?
